### PR TITLE
ON-6481 # Removed try/catch for jwt verification

### PR DESCRIPTION
**Pull request by mymattcarroll**

### CODE REVIEW

File: lib/verify-jwt.js [hunk 1]
```diff
@@ -1,31 +1,37 @@
...
+          reject(error)
+        } else {
+          resolve(decoded)
+        }
+      },
+    )
   })
+
+  return decoded
 }
```
Comment (Zaxist) [inline lib/verify-jwt.js:37]: I get why you did this, but it seems a bunch more convoluted?

Comment (mymattcarroll) [reply lib/verify-jwt.js:37]: Yeah, code is not a pretty, but only decoding the JWT once now.

Comment (mymattcarroll) [reply lib/verify-jwt.js:37]: Ill clean it up slightly :)

Comment (Zaxist) [reply lib/verify-jwt.js:37]: Alrighty

**Pull request merged**